src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -349,8 +349,48 @@ public void OnBindingStateChanged_NewConfiguration_EventsRaisedInCorrectOrder()
                 // Assert
                 // Different config so event should be raised
                 eventCounter.PreSolutionBindingChangedCount.Should().Be(1);
+                eventCounter.PreSolutionBindingUpdatedCount.Should().Be(0);
+                eventCounter.SolutionBindingChangedCount.Should().Be(1);
                 eventCounter.SolutionBindingUpdatedCount.Should().Be(0);
+
+                eventCounter.RaisedEventNames.Should().HaveCount(2);
+                eventCounter.RaisedEventNames[0].Should().Be("PreSolutionBindingChanged");
+                eventCounter.RaisedEventNames[1].Should().Be("SolutionBindingChanged");                    
+            }
+        }
+        
+        [TestMethod]
+        public void HandleBindingChange_NewConfiguration_EventsRaisedInCorrectOrder()
```

Comment:
This test seems to be a copy-past of OnBindingStateChanged_NewConfiguration_EventsRaisedInCorrectOrder.
Can we maybe extract the whole content in a method and provide the act as a parameter to the method?

Reply:
The event-based test will be removed once we get rid of the old binding window